tests/e2e/chain/config.go

```diff
@@ -130,7 +133,7 @@ func addAccount(path, moniker, amountStr string, accAddr sdk.AccAddress) error {
 	return genutil.ExportGenesisFile(genDoc, genFile)
 }
 
-func initGenesis(c *internalChain) error {
+func initGenesis(c *internalChain, votingPeriod time.Duration) error {
```

Comment:
In the future, this can get messy. We should revise this such that we can instead pass the entire module genesis state. But for now, this is OK.